exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -79,9 +91,14 @@
                   </span>
                 </td>
                 <td id="status" >${drillbit.getState()}</td>
+                <!-- #if (model.shouldShowAdminInfo() || !model.isAuthEnabled()) -->
                 <#if (model.shouldShowAdminInfo() || !model.isAuthEnabled()) && drillbit.isCurrent() >
                   <td>
-                      <button type="button" id="shutdown" onClick="shutdown($(this));"> SHUTDOWN </button>
+                      <button type="button" id="shutdown" onClick="shutdown($(this));"><span class="glyphicon glyphicon-off"></span> SHUTDOWN </button>
+                  </td>
+                <#elseif (model.shouldShowAdminInfo() || !model.isAuthEnabled()) && !drillbit.isCurrent() >
+                  <td>
```

Comment:
Done the change for checking for HTTPS. However, this commit just marginally fixes the UI, which (otherwise) should have a full revamp. I did attempt some of the details proposed in DRILL-6244, but the effort is not trivial.
For an unsecured setup, there is little reason to restrict the ability to shutdown multiple nodes, because you can always go directly and shut down those nodes without authentication. This simply makes it convenient to have a single UI window to achieve the same outcome.